docs/development/Building in Ubuntu.md

```diff
@@ -8,7 +8,9 @@ This document is tested and based on the latest Ubuntu 20.04 LTS release.
 ```
 sudo apt update
 sudo apt upgrade
-sudo apt install build-essential
+sudo apt install build-essential python clang-8 libblocksruntime-dev
+sudo update-alternatives --install /usr/bin/clang clang /usr/bin/clang-8 100
```

Comment:
I'd be careful with this - this has the potential to screw up any number of other projects that rely on a more current version of `clang` to be the default.
We should look into changing our build to explicitly use `clang-8` instead.

Reply:
Only unit tests are dependent on clang-8. Why is it not preferred to solve this errors so we are able to use current clang? Maybe there is a command line switch to solve it? Anyway I agree the using linux alternatives should be avoided.

For now people can find this in the Wiki (https://github.com/betaflight/betaflight/wiki/Known-Build-Environment-Problems#compiling-unit-tests-fails-with-errors). Just added your warning to it.

Reply:
I did not mean to not solve it. I just proposed that instead of adding an alternative for `clang` to `clang-8` that might interfere with other uses of `clang` on the system we instead change the tests to use `clang-8` as the name of the binary they call.

Reply:
See https://github.com/betaflight/betaflight/pull/10265.

Reply:
Fixed for Ubuntu 20.04. Only thing left is Python?